app/src/main/java/com/example/xpvehicles/activities/AddVehicleActivity.java

```diff
@@ -152,16 +146,18 @@ private void setAddVehicleOnClickListener() {
     }
 
     private void getPlaceId(String streetAddress, String city, String state) {
-        String base_url = "https://maps.googleapis.com/maps/api/geocode/json?";
-        String address = streetAddress + " " + city + " " + state;
+        final String GOOGLE_GEOCODING_API_BASE_URL = "https://maps.googleapis.com/maps/api/geocode/json?";
+        final String GOOGLE_GEOCODING_API_PARAMETER_KEY = "key";
+        final String GOOGLE_GEOCODING_API_PARAMETER_ADDRESS = "address";
```

Comment:
You would want to move this to class level. BTW, 

public class PhysicalConstants {
    private PhysicalConstants() { }  // Prevents instantiation

    private static final double AVOGADROS_NUMBER   = 6.02214199e23;
    private static final double BOLTZMANN_CONSTANT = 1.3806503e-23;
    private static final double ELECTRON_MASS      = 9.10938188e-31;
}